src/move_ref.rs

```diff
@@ -189,27 +207,59 @@ impl<'a, T> From<MoveRef<'a, T>> for Pin<MoveRef<'a, T>> {
 ///   pointee type to [`MaybeUninit<T>`].
```

Comment:
Just a note, on line 199 of the new file, you have a dangling "To" at the end of the sentence.  Github doesn't let me comment on unchanged lines.